packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -40,7 +40,7 @@ export interface IMultiStackedBarChartState {
 
 export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarChartProps, IMultiStackedBarChartState> {
   public static defaultProps: Partial<IMultiStackedBarChartProps> = {
-    barHeight: 16,
+    barHeight: 12,
```

Comment:
does the size of legend also need to be updated

Reply:
didn't find any guidance for legend size in the work item